### DISCUSSION TITLE
WebUI Login Banner

### DISCUSSION THREAD

**Discussion by out-n-about** [Security-Onion-Solutions/securityonion#2643] [Unsupported Versions] [answered]
Is there anyway to add a login banner to the WebUI Login page?  If so where are the files located for that page and how can I do that?

Thanks

**Comment by jertel** [comment, marked as answer]
Currently no, there is not a simple way to do this. I've created #2647 to track this as an enhancement.

**Reply by out-n-about** [reply]
Ok thanks!
